locmap/lib/locMapRESTAPI2.js

```diff
@@ -53,6 +53,99 @@ var LocMapRESTAPI2 = function() {
 
         callback(200, places);
     };
+    this.placeNameAlreadyInUse = function(user, placeName) {
+        for (var key in user.data.places) {
+            if (user.data.places[key].name === placeName) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Remove this and use restApi.placeNameAlreadyInUse instead since this is the same function